orderbook/openapi.yml

```diff
@@ -153,8 +153,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/FeeInformation"
-        400:
+        404:
           description: sellToken not existent
+  /api/v1/trades/:
+    get:
+      summary: Get existing Trades.
+      description: |
+        By default all trades are returned.
+        Queries can be refined by specifiying owner or order_uid.
+      parameters:
```

Comment:
not need to implemet now, but I would expect in the future:
- token: Either sell or buy token. Allows to see trades with one token
- I'm not adding the param for market, cause that one may be sth like `/markets/<market>/trades` 
- sellToken
- buyToken
- batchId: Trades included in a batch
- appId: Trades with a given appId
- fromDate
- toDate

Reply:
Created an issue #311 for these points. 

Note that the last one seems to be a request for orders... I assume you want to see trades for open and not open orders.